templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -655,9 +655,14 @@ export default {
               },
             })
           }
-          if (this.node.status == "draft") {
-            this.updateSelectedNode(id)
-          }
+        }
+        if (this.node.status == "draft") {
+          this.updateSelectedNode(id)
+          this.$router.push({
+            name: names.APP,
+            params: { nodeId: id },
+            query: this.$route.query,
+          })
```

Comment:
@jaskwong can you explain what this fixes exactly? What was the bug we were having before this fix?

Reply:
I think it was just to add the router push after updating the selected node, although it’s been a while since the other PR so maybe it’s not necessary anymore